src/shared/models/friends.model.ts

```diff
@@ -0,0 +1,8 @@
+import type mongoose from 'mongoose';
+import type { User } from './user.model.js';
+export interface Friends {
+    _id?:string
+    text:string
+    user: {type: mongoose.Types.ObjectId} | User;
```

Comment:
you can remove the mongoose stuff because when the information gets back from the db, we want it to take what shape ... you can just make this a user

Reply:
Also, I think Saba can make it a draft for the time and can work on making a friends collection with id, user, friends[] properties or other options discussed in the class today.

Reply:
I changed the friends model and schema as suggested with a property of id, user and friends.